src/Umbraco.Web/Trees/UrlHelperExtensions.cs

```diff
@@ -30,5 +33,39 @@ public static string GetMenuUrl(this UrlHelper urlHelper, Type treeType, string
             return actionUrl;
         }
 
+
+        public static string GetTreePathFromFilePath(this UrlHelper urlHelper, string virtualPath, string basePath = "")

```

Comment:
Should be Internal, we will need to move this elsewhere at some point since it actually has no reliance on UrlHelper but changing it internal for now will be fine.